### PR TITLE
File not found logging can now be error or info (fix #9)

### LINKED ISSUE #9: Logging "errors" that are really "info"
On running the tool for the first time, users are presented with a number of red "error" logs that are not really errors, such as:

```
INFO[0000] Validator started
ERRO[0000] open rrdp.json: no such file or directory
ERRO[0000] Error exploring file: open cache/rpki.afrinic.net/repository/AfriNIC.cer: no such file or directory
ERRO[0000] Error exploring file: open cache/rpki.apnic.net/repository/apnic-rpki-root-iana-origin.cer: no such file or directory
ERRO[0000] Error exploring file: open cache/rpki.arin.net/repository/arin-rpki-ta.cer: no such file or directory
ERRO[0000] Error exploring file: open cache/repository.lacnic.net/rpki/lacnic/rta-lacnic-rpki.cer: no such file or directory
ERRO[0000] Error exploring file: open cache/rpki.ripe.net/ta/ripe-ncc-ta.cer: no such file or directory
INFO[0000] Still exploring. Revalidating now
INFO[0000] Rsync sync rsync://rpki.arin.net/repository/arin-rpki-ta.cer
INFO[0001] Rsync sync rsync://rpki.ripe.net/ta/ripe-ncc-ta.cer
INFO[0002] Rsync sync rsync://rpki.afrinic.net/repository/AfriNIC.cer
INFO[0004] Rsync sync rsync://rpki.apnic.net/repository/apnic-rpki-root-iana-origin.cer
INFO[0007] Rsync sync rsync://repository.lacnic.net/rpki/lacnic/rta-lacnic-rpki.cer
ERRO[0010] Error exploring file: open cache/rpki.afrinic.net/repository/04E8B0D80F4D11E0B657D8931367AE7D/62gPOPXWxxu0sQa4vQZYUBLaMbY.mft: no such file or directory
ERRO[0010] Error exploring file: open cache/rpki.apnic.net/repository/838DB214166511E2B3BC286172FD1FF2/C5zKkN0Neoo3ZmsZIX_g2EA3t6I.mft: no such file or directory
ERRO[0010] Error exploring file: open cache/rpki.arin.net/repository/arin-rpki-ta/arin-rpki-ta.mft: no such file or directory
ERRO[0010] Error exploring file: open cache/repository.lacnic.net/rpki/lacnic/rta-lacnic-rpki.mft: no such file or directory
ERRO[0010] Error exploring file: open cache/rpki.ripe.net/repository/ripe-ncc-ta.mft: no such file or directory
INFO[0010] Still exploring. Revalidating now
```
As these files haven't been created yet (it's the first run) these really ought to be "info" priority, or "warning" at most. An error generally means something that is causing functionality to stop due to an unexpected issue that may not be able to be recovered from.

Would you consider changing these to a lower priority, and/or to change the message so that it indicates this is a recoverable state to be in?

### PR DESCRIPTION
**Pull request by lspgn**
cc @dotwaffle

### CODE REVIEW

**Comment by dotwaffle** [comment]
Thanks, am out in the states ATM, will try and review later today.

**Comment by ericat** [comment]
The output as of now:
screenshot 2019-03-07 at 15 08 08

File: sync/lib/fetch.go [hunk 1]
```diff
@@ -64,6 +65,21 @@ func (s *LocalFetch) GetFile(file *pki.PKIFile) (*pki.SeekFile, error) {
...
+        var contained bool
+        for _, pa := range s.PathAvailable {
+            contained = strings.Contains(file.Path, pa)
+            if contained {
+                break
+            }
+        }
+        if !contained {
+            if s.Log != nil {
+                s.Log.Infof("Error on repository not yet synchronized: %v", err)
```
Comment (dotwaffle) [inline sync/lib/fetch.go]: Why use the string "Error" here when you're only raising a log with priority "Info"?

Comment (lspgn) [reply sync/lib/fetch.go]: Fixed -> `Got %v but repository not yet synchronized`

**Pull request merged**